AppController/djinn.rb

```diff
@@ -1778,6 +1786,18 @@ def job_start(secret)
           scale_appservers_across_nodes()
         end
       }
+
+      # We want to print some statistic in the logs every hour or so, to
```

Comment:
This comment is too verbose. Please change it to something like:

"Print stats in the log recurrently; works as a heartbeat mechanism."